ee/codegen_integration/fixtures/simple_code_execution_node/code/display/workflow.py

```diff
@@ -19,7 +17,7 @@
 
 
 class WorkflowDisplay(VellumWorkflowDisplay[Workflow]):
-    workflow_display = WorkflowMetaVellumDisplayOverrides(
+    workflow_display = WorkflowMetaDisplay(
         entrypoint_node_id=UUID("d49107fe-1424-42ba-9413-9ab5ce398077"),
         entrypoint_node_source_handle_id=UUID("08d78489-ce80-4743-a22d-2d5f62b575ac"),
```

Comment:
This and the replace below are the end states we are working towards. Everything else is a precursor